src/main/java/com/mongodb/hibernate/internal/translate/mongoast/command/aggregate/AstProjectStage.java

```diff
@@ -16,10 +16,17 @@
 
 package com.mongodb.hibernate.internal.translate.mongoast.command.aggregate;
 
+import static com.mongodb.hibernate.internal.MongoAssertions.assertFalse;
+
 import java.util.List;
 import org.bson.BsonWriter;
 
 public record AstProjectStage(List<? extends AstProjectStageSpecification> specifications) implements AstStage {
+
+    public AstProjectStage {
+        assertFalse("Projection stage must have at least one specification", specifications.isEmpty());
```

Comment:
This static message provides no information that is not already clear from the code. Given that assertions can be violated only when there are bugs in the product, `AssertionError`s do not have a requirement to be understood without looking at the product code. Thus, having static error messages in `AssertionError`s is completely useless.

Let's remove the message and have only `assertFalse(specifications.isEmpty());`

Reply:
done